fractional-NFT/project/fractional-NFT-contract/src/data_structures.sw

```diff
@@ -0,0 +1,21 @@
+library data_structures;
+
+/// Contains information on the NFT this contract holds.
+pub struct NFTInfo {
+    /// The identity which may unlock and withdraw the NFT.
+    admin: Option<Identity>,
+    /// The contract which manages the NFT.
+    nft: ContractId,
```

Comment:
hm, so it's whatever minted the NFT? If that is the case then would it be better to rename this to `asset_id`? `nft_info.nft` isn't as intuitive as `nft_info.asset_id`